backend/index.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express')
+const app = express()
+app.use(express.json())
+const mongoose = require('mongoose')
+const connection_url = "mongodb+srv://newuser:newPassword@cluster0.w4wyn.mongodb.net/RecipeDB?retryWrites=true&w=majority"
+
+mongoose.connect(connection_url)
+.then(()=> console.log('Succesfully Connected'))
+.catch((error)=> console.log('Could not connect due to ${error}'))
+
+const Recipe = require("./models/recipeSchema")
+
+app.get('/api/recipe', async (req, res) => {
```

Comment:
It's definitely not mandatory, but putting a short, descriptive comment above an endpoint is a good practice that makes your backend code easier to read!